### PR TITLE
test(atomic): Add Cypress test for atomicFormat value component

### PR DESCRIPTION
**Pull request by lvu285**
https://coveord.atlassian.net/browse/KIT-834
- Test cover for atomic-format-number, atomic-format-currency and atomic-format-unit

### CODE REVIEW

File: packages/atomic/cypress/integration/facets-v1/numeric-facet/numeric-facet-assertions.ts [hunk 1]
```diff
@@ -129,3 +129,122 @@ function getEvenRangeValue(value: string) {
...
+    NumericFacetSelectors.valueLabel().as('facetAllValuesLabel');
+    cy.getTextOfAllElements('@facetAllValuesLabel').then((originalValues) => {
+      originalValues.forEach((e: string) => {
+        expect(e.split(' to ')[0]).contains('kg');
+        expect(e.split(' to ')[1]).contains('kg');
+        expect(e.split(' to ')[0].slice(-2)).eq('kg');
+      });
+    });
+  });
+}
```
Comment (nbuissoncoveo) [inline packages/atomic/cypress/integration/facets-v1/numeric-facet/numeric-facet-assertions.ts:170]: Shouldn't this be `hr` instead of `kg`?

**Review by nbuissoncoveo — APPROVED** [review]

File: packages/atomic/cypress/integration/facets-v1/numeric-facet/numeric-facet-assertions.ts (hunk 1 above)
Comment (lvu285) [reply packages/atomic/cypress/integration/facets-v1/numeric-facet/numeric-facet-assertions.ts:170]: nice catch, thanks, the title of the test should be 'kilogram' :)

File: packages/atomic/cypress/integration/facets-v1/numeric-facet/numeric-facet-assertions.ts [hunk 2]
```diff
@@ -129,3 +129,122 @@ function getEvenRangeValue(value: string) {
...
 }
+
+export function assertCurrencyFormat() {
+  it('should display correct currency format CA$', () => {
+    NumericFacetSelectors.valueLabel().as('facetAllValuesLabel');
+    cy.getTextOfAllElements('@facetAllValuesLabel').then((originalValues) => {
+      originalValues.forEach((e: string) => {
+        expect(e.split(' to ')[0]).contains('CA$');
+        expect(e.split(' to ')[1]).contains('CA$');
+        expect(e.split(' to ')[0].split('$')[0]).eq('CA');
```
Comment (samisayegh) [inline packages/atomic/cypress/integration/facets-v1/numeric-facet/numeric-facet-assertions.ts:140]: An alternative could be to split once, and reuse the values:

```
const [start, end] = e.split(' to ');
expect(start).contains('CA$');
expect(end).contains('CA$');
```

**Review by samisayegh — APPROVED** [review]

**Review by btaillon — APPROVED** [review]

**Review by ThibodeauJF — APPROVED** [review]

**Review by nbuissoncoveo — APPROVED** [review]

**Comment by ThibodeauJF** [comment]
Build passed on my other PR with the same commits so merging it!

**Pull request merged**